Description: do not overwrite version if we are not running setup.py not in a hg checkout
Author: Ludovico Cavedon <ludovico.cavedon@gmail.com>

Index: tortoisehg/setup.py
===================================================================
--- tortoisehg.orig/setup.py	2009-11-21 02:01:12.000000000 -0800
+++ tortoisehg/setup.py	2009-11-21 02:01:16.000000000 -0800
@@ -172,6 +172,8 @@
     elif len(l) == 1: # no tag found
         cmd = 'hg parents --template {latesttag}+{latesttagdistance}-'
         version = os.popen(cmd).read() + l[0]
+    else:
+        version = "unknown"
     if version.endswith('+'):
         version += time.strftime('%Y%m%d')
 except OSError:
